addon/components/hyper-table/cell-renderers/date.js

```diff
@@ -27,5 +27,21 @@ export default Component.extend({
     }
 
     return _date.format(this._dateFormat);
-  })
+  }),
+  isEditing: false,
+
+  actions: {
+    toggleEditing() {
+      this.set("isEditing", true);
+      this.flatpickrRef.open();
+    },
+    onCalendarClose() {
+      this.set("isEditing", false);
+    },
+    updateDate(date) {
```

Comment:
what happen if no date selected ?

Reply:
This will only be fired when clicking on a date, it's not possible not to select a date (a user can close the calendar without selecting a date, in that case it won't be fired at all)